pkg/meta/meta.go

```diff
@@ -423,6 +423,8 @@ func AllowsPropagationTo(from metav1.Object) map[types.NamespacedName]bool {
 
 // IsPaused returns true if the object has the AnnotationKeyReconciliationPaused
 // annotation set to `true`.
+// Deprecated: Use reconciler.IsPaused instead to check if the Crossplane
+// managed resource is paused as it includes the check for management policies.
```

Comment:
Could we make this call `reconciler.IsPaused`? My guess is that might not be possible/desirable due to an import cycle.

Reply:
yeah there is an import cycle. Plus I am not sure if the meta package should be aware of management policies.